lms/djangoapps/courseware/views/index.py

```diff
@@ -183,7 +183,7 @@ def _verify_position(self):
         """
         if self.position is not None:
             try:
-                int(self.position)
+                self.position = max(int(self.position), 1) # test change
```

Comment:
^ To that note, in the initial description, replace `TNL-4408` with `[TNL-4408](https://openedx.atlassian.net/browse/TNL-4408)`